bin/env_variables_all.txt

```diff
@@ -1,77 +1,78 @@
 CCD_GW_SERVICE_NAME=ccd_gw
 CCD_GW_SERVICE_SECRET=AAAAAAAAAAAAAAAC
+CCD_IMPORT_AUTOTEST_EMAIL=auto.test.cnp@gmail.com
+CCD_IMPORT_AUTOTEST_PASSWORD=Pa55word11
+CCD_PRIVATE_CASEWORKER_EMAIL=auto.test.cnp+private@gmail.com
+CCD_PRIVATE_CASEWORKER_PASSWORD=Pa55word11
 CCD_PRIVATE_CASEWORKER_SOLICITOR_EMAIL=auto.test.cnp+solc@gmail.com
 CCD_PRIVATE_CASEWORKER_SOLICITOR_PASSWORD=Pa55word11
 CCD_PRIVATE_CROSS_CASE_TYPE_CASEWORKER_EMAIL=auto.test12.cnp+private@gmail.com
 CCD_PRIVATE_CROSS_CASE_TYPE_CASEWORKER_PASSWORD=Pa55word11
 CCD_PRIVATE_CROSS_CASE_TYPE_SOLICITOR_EMAIL=auto.test12.cnp+solc@gmail.com
 CCD_PRIVATE_CROSS_CASE_TYPE_SOLICITOR_PASSWORD=Pa55word11
-CCD_PRIVATE_CASEWORKER_EMAIL=auto.test.cnp+private@gmail.com
-CCD_PRIVATE_CASEWORKER_PASSWORD=Pa55word11
 CCD_RESTRICTED_CASEWORKER_EMAIL=auto.test.cnp+senior@gmail.com
 CCD_RESTRICTED_CASEWORKER_PASSWORD=Pa55word11
 CCD_RESTRICTED_CROSS_CASE_TYPE_CASEWORKER_EMAIL=auto.test12.cnp+senior@gmail.com
 CCD_RESTRICTED_CROSS_CASE_TYPE_CASEWORKER_PASSWORD=Pa55word11
-DM_STORE_BASE_URL=http://dm-store:8080
-CCD_DM_DOMAIN=http://dm-store:8080
 DATA_STORE_IDAM_KEY=AAAAAAAAAAAAAAAB
+DATA_STORE_S2S_AUTHORISED_SERVICES=ccd_gw,ccd_gateway,fpl_case_service,ccd_data,ccd_ps,aac_manage_case_assignment
+DATA_STORE_TOKEN_SECRET=iuasbcuasdcbasdgcasdgcuysachjsacyasdgjcgasdj
+DB_PASSWORD=Pa55word11
+DB_USERNAME=ccd
+DEFINITION_STORE_DB_USE_SSL=false
 DEFINITION_STORE_HOST=http://localhost:4451
 DEFINITION_STORE_IDAM_KEY=AAAAAAAAAAAAAAAA
+DEFINITION_STORE_S2S_AUTHORISED_SERVICES=ccd_data,ccd_gw,ccd_admin
+DM_STORE_BASE_URL=http://dm-store:8080
 ELASTIC_SEARCH_ENABLED=true
 ELASTIC_SEARCH_HOST=localhost
 ELASTIC_SEARCH_HOSTS=http://localhost:9200
 ELASTIC_SEARCH_PORT=9200
 ELASTIC_SEARCH_SCHEME=http
+IDAM_KEY_CCD_ADMIN=AAAAAAAAAAAAAAAD
+IDAM_KEY_CCD_DATA_STORE=AAAAAAAAAAAAAAAB
+IDAM_KEY_CCD_DEFINITION_DESIGNER=AAAAAAAAAAAAAAAE
+IDAM_KEY_CCD_DEFINITION_STORE=AAAAAAAAAAAAAAAA
+IDAM_KEY_CCD_GATEWAY=AAAAAAAAAAAAAAAC
+IDAM_KEY_DM_STORE=AAAAAAAAAAAAAAAA
+IDAM_KEY_FPL_CASE_SERVICE=AABBCCDDEEFFGGHH
 IDAM_OAUTH2_CLIENT_ID=ccd_gateway
 IDAM_OAUTH2_CLIENT_SECRET=ccd_gateway_secret
 IDAM_S2S_URL=http://localhost:4502
 IDAM_URL=http://localhost:5000
-IDAM_KEY_CCD_GATEWAY=AAAAAAAAAAAAAAAC
-OAUTH2_CLIENT_ID=ccd_gateway
+IDAM_USER_URL=http://localhost:5000
+OAUTH2_CLIENT_CCD_ADMIN=IIIIIIIIIIIIIIII
 OAUTH2_CLIENT_CCD_GATEWAY=ccd_gateway_secret
-USER=someuser
+OAUTH2_CLIENT_ID=ccd_gateway
 OAUTH2_CLIENT_SECRET=ccd_gateway_secret
 OAUTH2_REDIRECT_URI=http://localhost:3451/oauth2redirect
 S2S_URL=http://localhost:4502
 TEST_URL=http://localhost:4452
-DATA_STORE_S2S_AUTHORISED_SERVICES=ccd_gw,ccd_gateway,fpl_case_service,ccd_data,ccd_ps,aac_manage_case_assignment
-DEFINITION_STORE_S2S_AUTHORISED_SERVICES=ccd_data,ccd_gw,ccd_admin
-USER_PROFILE_S2S_AUTHORISED_SERVICES=ccd_data,ccd_definition,ccd_admin
-DATA_STORE_TOKEN_SECRET=iuasbcuasdcbasdgcasdgcuysachjsacyasdgjcgasdj
+USER=someuser
 USER_PROFILE_HOST=http://ccd-user-profile-api:4451
```

Comment:
Is this the correct port number or should it be pointing at 4453?

Reply:
It has been inherently incorrect. just fixed.